.teamcity/settings.kts

```diff
@@ -907,6 +908,7 @@ object RunCalypsoE2eDesktopTests : BuildType({
 				export LIVEBRANCHES=true
 				export NODE_CONFIG_ENV=test
 				export TEST_VIDEO=true
+				export PLAYWRIGHT_BROWSERS_PATH=0
```

Comment:
Without this line, the environment variable does not carry over from the setup step and thus Playwright looks for the browser binaries in the default directory.